app/src/main/java/com/mobilecoronatracker/ui/cumulatedreport/CumulatedReportViewModel.kt

```diff
@@ -39,6 +46,7 @@ class CumulatedReportViewModel(private val covidDataRepo: CovidDataRepo) : ViewM
         cases.postValue(data.cases.toString())
         deaths.postValue(data.deaths.toString())
         recovered.postValue(data.recovered.toString())
+        isRefreshing.postValue(false)
```

Comment:
This will stop progress indicator with first received item?

Reply:
It will stop progress on every received item, as an indicator can be set to true again by `refreshData`